src/main/java/com/dnd/namuiwiki/jwt/JwtProvider.java

```diff
@@ -0,0 +1,76 @@
+package com.dnd.namuiwiki.jwt;
+
+import com.dnd.namuiwiki.common.exception.ApplicationErrorException;
+import com.dnd.namuiwiki.common.exception.ApplicationErrorType;
+import io.jsonwebtoken.*;
+import io.jsonwebtoken.io.Decoders;
+import io.jsonwebtoken.security.Keys;
+import org.springframework.beans.factory.InitializingBean;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import javax.crypto.SecretKey;
+import java.util.Date;
+
+@Component
+public class JwtProvider implements InitializingBean {
+
+    @Value("${jwt.key}")
+    public String key;
+
+    @Value("${jwt.valid-time.access-token}")
+    private long accessTokenValidTime;
+
+    @Value("${jwt.valid-time.refresh-token}")
+    private long refreshTokenValidTime;
+    private final String WIKI_ID = "WIKI_ID";
+    private SecretKey secretKey;
+
+    @Override
+    public void afterPropertiesSet() {
+        byte[] keyBytes = Decoders.BASE64.decode(key);
+        this.secretKey = Keys.hmacShaKeyFor(keyBytes);
+    }
+
+    public String createAccessToken(String wikiId) {
+        return Jwts.builder()
+                .header()
+                    .add("typ", "JWT")
+                    .add("alg", "HS256")
+                .and()
+                .claims()
+                    .add(WIKI_ID, wikiId)
+                .and()
+                .expiration(new Date(new Date().getTime() + accessTokenValidTime))
+                .signWith(secretKey)
+                .compact();
+    }
+
+    public String createRefreshToken() {
+        return Jwts.builder()
+                .header()
+                .add("typ", "JWT")
+                .add("alg", "HS256")
+                .and()
+                .expiration(new Date(new Date().getTime() + refreshTokenValidTime))
+                .signWith(secretKey)
+                .compact();
+    }
```

Comment:
claims와 validTime으로 토큰 빌드하는 메소드를 분리하면 나중에 페이로드에 값이 추가되었을 때 더 유연하게 대처할 수 있을 것 같아요!

```suggestion
    public String createAccessToken(String wikiId) {
        Claims claims = Jwts.claims()
                .subject("accessToken")
                .add(WIKI_ID, wikiId)
                .build();
        return createToken(refreshTokenValidTime, claims);
    }

    public String createRefreshToken() {
        Claims claims = Jwts.claims()
                .subject("refreshToken")
                .build();
        return createToken(refreshTokenValidTime, claims);
    }

    private String createToken(Long validTime, Claims claims) {
        Date now = new Date();
        return Jwts.builder()
                .issuedAt(now)
                .header()
                .add("typ", "JWT")
                .add("alg", "HS256")
                .and()
                .claims()
                .add(claims)
                .and()
                .expiration(new Date(now.getTime() + validTime))
                .signWith(secretKey)
                .compact();
    }
```